### PR TITLE
Allow for environment setup through a `.env` file

### PR DESCRIPTION
**Pull request by hannestyden**
This change allows for setting up the run-time environments using a `.env` file by adding a dependency on the `dotenv` gem.
It also adds the required configuration to the environment to get the specs pass.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+export SOUNDCLOUD_CLIENT_ID=69e93cf2209402f6f3137a6452cf498f
```
Comment (sferik) [inline .env:1]: Should this be committed?

Comment (hannestyden) [inline .env:1]: It already is. :wink:
I copied it from 94dd4daa7b0b0886574a127577072686f4986d70 , so it was already compromised.

Comment (sferik) [inline .env:1]: Okay, but we should fix that and generate a new dev :key:, no?

Comment (hannestyden) [inline .env:1]: Yeah, `app/assets/javascripts/plays.js` should be updated too.
Don't know how to get the acceptance tests to pass without it though ...

Comment (hannestyden) [inline .env:1]: ... or maybe the acceptance tests would work with any client id.

Also realized that this is the `client_id` and not the `client_secret`.

**Comment by starkcoffee** [comment]
Gonna merge and fix this.

**Pull request merged**